.gitignore

```diff
@@ -32,9 +32,6 @@ vendor
 # mkdocs folder
 mkdocs
 
-# airgap-push script directories
-kcm-airgap
-
 # Exclude CAPI operator manifests
 templates/provider/*/files
 # But keep kcm-templates
```

Comment:
This is also related to airgap (ignore collected manifests)
Last line as well, but I can't highlight it for some reason